RLTest/redis_std.py

```diff
@@ -187,7 +188,7 @@ def createCmdArgs(self, role):
 
         if self.port > -1:
             if self.useTLS:
```

Comment:
```suggestion
            if self.useTLS or self.dualTLS:
```

Reply:
I want to have the port duality only upon setting both `useTLS` and `dualTLS` (due to the certificates and keys needed)